submissions/vl-shevchenko/html-css-popup/index.html

```diff
@@ -0,0 +1,59 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="UTF-8">
+        <meta http-equiv="X-UA-Compatible" content="IE=edge">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <link rel="stylesheet" href="style.css">
+        <link rel="icon" href="popup-icons2/favicon.png" type="images/png">
+        <title>Hooli-style Popup</title>
+    </head>
+    <body>
+        <header>
+            <nav>
+                <div>                                 
+                    <div class="conteiner_menu">                   
+                        <ul class="conteiner__unlist">
+                            <li class="list"><a href="#">Mail</a></li>
+                            <li class="list"><a href="#">Images</a></li>
+                            <li>
+                                <input tabindex="-1" class="checkbox_knopka" type="checkbox" id="checkbox_knopka">
+                                <label class="list" for="checkbox_knopka">                               
+                                    <a tabindex="0"><img src="popup-icons2/01.png" alt=""></a>
+                                </label>         
```

Comment:
`tabindex="-1"` don't allow to get browser focus and this button is not accessible from the keyboard. Also, `tabindex="0"` makes no sense on interactive elements.